src/Form/SettingsForm.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Drupal\ezcontent\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class SettingsForm.
+ *
+ * @package Drupal\ezcontent\Form
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'ezcontent.settings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('ezcontent.settings');
+
+    $form['revision'] = [
+      '#type'  => 'fieldset',
+      '#title' => 'Hide Entity Revision Field',
+      '#open'  => TRUE,
+    ];
+
+    $form['revision']['media'] = [
+      '#type'  => 'fieldset',
```

Comment:
Please make it collapsible and default as `collapsed`.
Check FVM settings at path `admin/structure/display-modes/view/fvm`.